index.js

```diff
@@ -118,14 +118,24 @@ module.exports = (SECRET_KEY) => {
       .set('Authorization', 'Signature ' + hash)
   }
 
+  function fromPromise (req, res, next) {
+    res.fromPromise = promise => {
+      return promise
+        .then(result => res.status(result.status).send(result.body))
```

Comment:
I'd rather this just send back `result`, so that you aren't forced to use a result or an error that has a `body` property.

I take it the ssbot stuff uses body in its results and errors?

Reply:
Yeah, it's used by the ssbot stuff. The primary purpose of this function is to split the result into a status and a body that get returned in the response.